src/events/cd-event-ticket.vue

```diff
@@ -1,14 +1,19 @@
 <template>
-  <div class="cd-event-tickets">
-    <div class="cd-event-tickets__head"></div>
-    <div class="cd-event-tickets__ticket">
-      <span class="cd-event-tickets__name"><span class="cd-event-tickets__name-for">Name:</span>{{ user.firstName }} {{ user.lastName }}</span>
-      <div class="cd-event-tickets__ticket-selector"> 
-        <multiselect v-model="tickets" :options="ticketsOptions" group-label="name" group-values="tickets" :multiple="true" :searchable="false" :group-select="false" :placeholder="$t('Select tickets')" track-by="id" label="name" @close="onBlur" @open="onFocus" :data-vv-name="`tickets-${user.id}`" v-validate="'required'"></multiselect>
+  <div class="cd-event-tickets" :class="{ 'before-disabled': notAttending }">
+    <div class="cd-event-tickets__head" :class="{ 'head-disabled': notAttending }"></div>
```

Comment:
If you follow the above advice, you won't need this class anymore. The `--disabled` modifier on the root element will allow you to style everything you need.